package.json

```
@@ -4,18 +4,21 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "build": "babel src --out-dir dist",
+    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "db:roll": "sequelize db:migrate",
+    "db:undo": "sequelize db:migrate:undo:all",
```

We can add a `db:setup` script
```suggestion
    "db:undo": "sequelize db:migrate:undo:all",
    "db:setup": "npm run db:migrate && sequelize db:seed:all",
```